bindings/wasm/examples/node/revocation.js

```diff
@@ -35,7 +35,7 @@ async function revoke(clientConfig) {
     issuer.doc.removeMethod(DID.parse(issuer.doc.id.toString()+"#newKey"));
     issuer.doc.previousMessageId = issuer.nextMessageId;
```

Comment:
`issuer.nextMessageId` is also a receipt. It might be better to explicitly rename all existing `messageId` instances to `messageReceipt` to avoid confusion.